### DISCUSSION TITLE
Jolt Physics Engine

### DISCUSSION THREAD

**Discussion by ulvido** [bevyengine/bevy#16550] [Q&A] [answered]
Is there a way to use [Jolt Physics](https://github.com/jrouwe/JoltPhysics) with Bevy Engine like Rapier Physics?

**Comment by Jondolf** [comment, marked as answer]
I'm not aware of any Jolt integrations for Bevy, and there aren't even many [Jolt bindings for Rust](https://crates.io/search?q=jolt) in general, certainly nothing complete. So, no, unless you're willing to write your own integration.
